test/integration/fixtures/blocks/core__social-links__deprecated-1.html

```diff
@@ -0,0 +1,7 @@
+<!-- wp:social-links {"customIconColor":"#ffffff","iconColorValue":"#ffffff","customIconBackgroundColor":"#3962e3","iconBackgroundColorValue":"#3962e3","className":"has-icon-color"} -->
+<ul class="wp-block-social-links has-icon-color has-icon-background-color items-justified-space-between">
```

Comment:
How can this block have `item-justified-*` classname without having the `itemsJustification` attribute?

Reply:
I guess that attribute has never been registered (was broken)

Reply:
Exactly.
>I've [found a bug](https://github.com/WordPress/gutenberg/issues/34003) in `Social Icons` block regarding `itemsJustification` attribute which was introduced in https://github.com/WordPress/gutenberg/pull/28980 and wasn't declared in block.json. With this PR the issue will be resolved as we remove its usage.